sys/Makefile.include

```diff
@@ -69,4 +69,10 @@ ifneq (,$(filter arduino,$(USEMODULE)))
     include $(RIOTBASE)/sys/arduino/Makefile.include
 endif
 
+ifneq (,$(filter printf_float,$(USEMODULE)))
+    ifdef USE_NANO_SPECS
```

Comment:
IMO `ifeq (1,$(USE_NANO_SPECS))` is more flexible since we can set USE_NANO_SPECS=0 if we don't want it as well.